easybuild/easyconfigs/x/X11/X11-20180128-GCCcore-6.4.0.eb

```diff
@@ -0,0 +1,216 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20180128'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+
+source_urls = [
+    XORG_LIB_SOURCE,
+    XORG_PROTO_SOURCE,
+    'http://xcb.freedesktop.org/dist/',
+    'http://xkbcommon.org/download/',
+    'http://cgit.freedesktop.org/xorg/util/macros/snapshot',
+    XORG_DATA_SOURCE + '/xkeyboard-config',
+]
+checksums = [
```

Comment:
This is not being used for the listed components, checksums need to be specified on a per-component basis...

Unfortunately, `--inject-checksums` doesn't support injecting checksums for components yet... :(